### PR TITLE
🔀🌐 [fr_FR] update translations for Core

### PR DESCRIPTION
**Pull request by deoxis9001**
**What kind of update does this PR provide?** *Please check*

- [x] new translations / updated translations / translation fixes
- [ ] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**Which language(s) did you add/update translations for?**
*use the xx_XX notation for exact language!*
* xx_XX
* yy_YY
* ...

**What is included in your update?**
* new translations for xyz
* updated translations for abc
* fixed bug foo
* ...

**Additional notes**
Please add any further notes here!

### CODE REVIEW

File: src/i18n/fr_FR/buildings.ts [hunk 1]
```diff
@@ -64,6 +64,20 @@ export default {
...
                 duration: '5 jours',
             },
+            {
+                caption: 'Grande caserne de pompiers',
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[0]?.length ?? 0) / 10),
```
Comment (Suchty112) [inline src/i18n/fr_FR/buildings.ts:74]: In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

File: src/i18n/fr_FR/buildings.ts [hunk 2]
```diff
@@ -114,7 +128,22 @@ export default {
...
-        extensions: [],
+        extensions: [
+            {
+                caption: "Grande station d'ambulance",
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[2]?.length ?? 0) / 10),
```
Comment (Suchty112) [inline src/i18n/fr_FR/buildings.ts:139]: In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

File: src/i18n/fr_FR/buildings.ts [hunk 3]
```diff
@@ -298,6 +340,35 @@ export default {
...
                 9
             ),
+            {
+                caption: 'Grand poste de police',
+                credits: 1_000_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[6]?.length ?? 0) / 10),
```
Comment (Suchty112) [inline src/i18n/fr_FR/buildings.ts:350]: In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

File: src/i18n/fr_FR/buildings.ts [hunk 4]
```diff
@@ -298,6 +340,35 @@ export default {
...
+                cannotDisable: true,
+            },
+            {
+                caption: 'Grande prison',
+                credits: 200_000,
+                coins: 50,
+                duration: '7 jours',
+                maxExtensionsFunction: (
+                    buildingsByType: Record<number, Building[]>
+                ): number => Math.floor((buildingsByType[6]?.length ?? 0) / 10),
```
Comment (Suchty112) [inline src/i18n/fr_FR/buildings.ts:364]: In the case of the fire department, police and ambulance extensions, the small stations also count in the count. you have now only added the normal stations to the count here.

**Review by Suchty112 — CHANGES_REQUESTED** [review]

File: src/i18n/fr_FR/buildings.ts (hunk 4 above)
Comment (deoxis9001) [reply src/i18n/fr_FR/buildings.ts]: ok thanks I'll send the corrections

File: src/i18n/fr_FR/buildings.ts (hunk 2 above)
Comment (deoxis9001) [reply src/i18n/fr_FR/buildings.ts]: ok thanks I'll send the corrections

File: src/i18n/fr_FR/buildings.ts (hunk 1 above)
Comment (deoxis9001) [reply src/i18n/fr_FR/buildings.ts]: ok thanks I'll send the corrections

**Review by Suchty112 — APPROVED** [review]
lgtm after the changes

**Pull request merged**